src/css/objects/_media.scss

```diff
@@ -245,172 +241,216 @@
 // We're bending some BEM rules here by having 1-deep nested selectors, but in
 // exchange for that we get a single common markup pattern for media objects.
 
-.media--last {
-  margin-bottom: 0;
-}
-
-.media--new-and-unread {
-
-  .media__headline {
-    position: relative;
-
-    &::before {
-      background-color: $color-primary;
-      border-radius: 50%;
-      content: '';
-      display: block;
-      height: 0.5rem;
-      left: ($base-line-height / -2);
-      position: absolute;
-      top: 1rem;
-      width: 0.5rem;
-    }
+.o-media--sm {
+  align-items: center;
+  display: flex;
+  margin: 0 0 $base-line-height;
 
+  .o-media__headline, .o-media__title {
+    margin: 0;
   }
 
-  .media__indicator-visited {
-    display: none;
+  > .o-media__figure {
+    margin-right: 1.125rem;
+    max-width: 5rem;
+    float: left;
+    margin-right: 3.55505%;
+    width: 4.5rem;
   }
 
+  .o-media__bylineText {
+    border-bottom: 4px solid $color-gray-light;
+    padding-bottom: 0.5rem;
+  }
 }
 
-.media--hp {
-  box-shadow: inset 17rem 0 0 $color-gray-lightest;
-  padding: $base-line-height 0 $base-line-height 19rem;
+.o-media--med {
+  // box-shadow: inset 17rem 0 0 $color-gray-lightest;
+  // padding: $base-line-height 0 $base-line-height 19rem;
   position: relative;
 
+  //NOTE: Re-add the following padding as part of the CSS specific to the homepage rather than here.
+  // padding-top: 4.25rem
+
+  .o-media__figure {
+    margin-bottom: 1.375rem;    
+  }
+
   @include media-max($media-tablet) {
     border-top: 1px solid $color-gray-light;
     box-shadow: none;
```

Comment:
Now that a box shadow isn't being set on the `.o-media--med` class, it doesn't need to get reset to `0` here.